Sources/StreamChatUI/Components.swift

```diff
@@ -106,6 +106,9 @@ public struct Components {
     // The view that displays a banner to show the count of messages
     public var messagesCountDecorationView: ChatMessagesCountDecorationView.Type = ChatMessagesCountDecorationView.self
 
+    // A view that displays a badge with text. Usually to display unread counts in a red badge.
+    public var badgeView: BadgeView.Type = BadgeView.self
```

Comment:
What's the scope of this badge view? Since we have few places where it can appear.

Reply:
What do you mean by the scope? 🤔 

Basically, previous we had only `ChatChannelUnreadCountView`, and we need this view for the `ThreadList` as well. So I extract the common parts to `badgeView` and created a new `ChatThreadUnreadCountView`.

Reply:
yeah, you answered my question 😄